app/src/main/java/com/example/mycleaninglog/MainActivity.kt

```diff
@@ -27,11 +27,7 @@ class MainActivity : ComponentActivity() {
                             .background(MaterialTheme.colors.background)
                             .padding(24.dp)
                     ) {
-                        //calls the 1st level expandable cards. hardcoded because they should never change. These make up the main menu
-                            ExpandableCard(title = "Rooms")
-                            ExpandableCard(title = "Common Tasks")
-                            ExpandableCard(title = "Upcoming Tasks")
-
```

Comment:
Removed these three calls and replaced it with the function HomeScreen that your group created to make these three calls. I assumed your group planned on calling this since it was already created.